tests/unit/test_mcp_gateway.py

```
@@ -2883,27 +2903,46 @@ async def test_400_for_non_hex_id(self, router_with_registry):
         assert resp.json() == {"error": "invalid_request"}
 
     @pytest.mark.asyncio
-    async def test_audit_log_no_reason_on_failure(self, router_with_registry, capsys):
-        app, _registry, _auth, _handshake = router_with_registry
+    async def test_audit_log_includes_reason_on_rejection(self, router_with_registry, capsys):
+        app, registry, _auth, handshake = router_with_registry
```

Ruff (0.15.12)

[warning] Unpacked variable `handshake` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)